recipes/libnfs/all/conanfile.py

```diff
@@ -59,6 +69,8 @@ def package(self):
         copy(self, "LICENCE*.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         cmake = CMake(self)
         cmake.install()
+        if Version(self.version) >= "6.0.2" and self.settings.os == "Windows" and self.options.shared:
+            rename(self, os.path.join(self.package_folder, "bin", "nfs.dll"), os.path.join(self.package_folder, "bin", "libnfs.dll"))
```

Comment:
This is odd - on Windows, consumers will link against the import library (`.lib`), which embeds the name of the expected `.dll` file -  does renaming the DLL not break at runtime because the correct DLL cannot be found?

Reply:
@jcar87 
Agreed.
However, in 6.0.2, when I build a shared library on Windows, the test_package fails to run due to lack of dll.
I will look into the reason and report back to upstream if necessary.

Reply:
Hi @toge ! Thanks so much for looking into it - any updates? anything we can do?

Reply:
@jcar87 
Upstream updated on this matter.
https://github.com/sahlberg/libnfs/commit/87c2b2a3dc605d4c7da932b2f88b002df722642a

I created the patch file about it instead of renaming dll file name.
Could you review it again?